### DISCUSSION TITLE
Does this server supports streaming KLV?

### DISCUSSION THREAD

**Discussion by gediminasgu** [bluenviron/mediamtx#435] [Questions] [answered]
Hi,

That is great peace of software. It was very easy to install and run. It works perfectly.
Does it support streaming KLV? I tried to pass MPEG-TS file but on another end, I got video only. Here is my command:
```
ffmpeg -re -stream_loop -1 -i video.ts -c copy -f rtsp rtsp://127.0.0.1:8554/mystream
```

Also, I tried to map streams by adding `-map: 0:0 -map 0:1` but got:
```
Could not write header for output file #0 (incorrect codec parameters ?): Server returned 400 Bad Request
```

Thank you for your great work!

**Comment by gediminasgu** [comment, marked as answer]
Ah, never mind. I see it's a duplicate of https://github.com/aler9/rtsp-simple-server/discussions/357

**Reply by aler9** [reply]
The problem is that ffmpeg is not able to stream KLV with RTP/RTSP, while the server is theoretically able to.

[a comment by an automated account is not shown]
